mono/eglib/gunicode.c

```diff
@@ -88,7 +88,7 @@ g_unichar_type (gunichar c)
 	/* since the argument is UTF-16, we cannot check beyond FFFF */
 
 	/* It should match any of above */
-	return 0;
+	return g_cast (0); // G_UNICODE_CONTROL
```

Comment:
This seems like an abuse of `g_cast`.  You have to write something here and all the types are known, so just write the explicit cast.

Reply:
“No change to C.” Useful approach?